python/CRUD-examples/delete/reset_data.py

```diff
@@ -0,0 +1,57 @@
+#Code Sample from the tutorial at https://learncodeshare.net/2015/07/09/delete-crud-using-cx_oracle/
+#  section titled "Resetting the data"
+#The following resets the data for use with the update section
+#For both tables:
+#  Table data is removed.
+#  The identity column is set to start with the id after the starting data.
+#  Using the executemany function an array of starting data is inserted into the table.
+
+import cx_Oracle
+import os
+connectString = os.getenv('db_connect')
+con = cx_Oracle.connect(connectString)
+cur = con.cursor()
+
+# Delete rows
+statement = 'delete from lcs_pets'
+cur.execute(statement)
+
+# Reset Identity Coulmn
+statement = 'alter table lcs_pets modify id generated BY DEFAULT as identity (START WITH 8)'
+cur.execute(statement)
+
+# Delete rows
+statement = 'delete from lcs_people'
+cur.execute(statement)
+
+# Reset Identity Coulmn
+statement = 'alter table lcs_people modify id generated BY DEFAULT as identity (START WITH 8)'
+cur.execute(statement)
+
+# Insert default rows
+rows = [(1, 'Bob', 35, 'I like dogs'),
+        (2, 'Kim', 27, 'I like birds'),
+        (3, 'Cheryl', 23, 'I like horses'),
+        (4, 'Bob', 27, 'I like rabbits'),
+        (5, 'Stacey', 45, 'I like snakes'),
+        (6, 'Pete', 23, 'I like cats'),
+        (7, 'Pat', 36, 'I like dogs')]
+cur.bindarraysize = 2
+cur.setinputsizes(int, 20, int, 100)
+cur.executemany("insert into lcs_people(id, name, age, notes) values (:1, :2, :3, :4)", rows)

```

Comment:
I've reproduced the CRUD tutorials in a couple other languages.  I'm trying to just use features that work in each language.  I'm not sure batcherrors will be an option in all of the languages I plan to use.